docs/Installing_IML_on_HPC_Storage_Sandbox.md

```diff
@@ -0,0 +1,143 @@
+```
+ct7-mds[1,2].lfs.local,ct7-oss[1,2].lfs.local
+```
+This will take some time (around 20 to 30 minutes) but all four servers should add successfully.
+
+## Configuring Interfaces
+Once all servers have been added, each server will need to know which interface should be assigned the lustre network. Navigate to each server detail page by clicking on the server link. Scroll to the bottom of the server detail page where you will see a list of network interfaces. Click on the `Configure` button and you will be given the option to change the network driver and the network for each interface. 
+
+The vagrant file indicates that the lustre network will run on 10.73.20.x. If `Lustre Network 0` is specified for a different IP address, you will need to change its interface to `Not Lustre Network` and update the network for 10.73.20.x to use `Lustre Network 0`. It is very important that Lustre Network 0 is specified on the correct interface; otherwise, creating a filesystem will fail. Make sure that all servers have been updated where appropriate.
+
+## Creating a Filesystem
+To create a filesystem, simply navigate to `Configure->File Systems` and click the `Create` button. Make the following selections:
+- Management Target / MGS -> mds1 (512 MB)
+- Metadata Target / MDS -> mds2
+- Object Storage Targets -> Select ALL OSS nodes
+
+After the selections have been made, click the button to create the filesystem. If you have any issues creating the filesystem there is a good chance that the interface for 10.73.20.x is not assigned to Lustre Network 0. If this happens, stop the filesystem and update the interfaces accordingly. 
+
+## Setting up Clients
+In your vagrant folder, run the following script to prepare both client c1 and c2:
+```
+vagrant sh -c '\
+sudo yum -y install epel-release && \
+sudo yum-config-manager --add-repo https://copr.fedorainfracloud.org/coprs/managerforlustre/manager-for-lustre/repo/epel-7/managerforlustre-manager-for-lustre-epel-7.repo && \
+sudo yum-config-manager --add-repo http://mirror.centos.org/centos/7/extras/x86_64/ && \
+sudo ed <<EOF /etc/yum.repos.d/mirror.centos.org_centos_7_extras_x86_64_.repo
+/enabled/a
+gpgcheck=1
+gpgkey=http://mirror.centos.org/centos/RPM-GPG-KEY-CentOS-7
+.
+wq
+EOF && \
+sudo yum-config-manager --add-repo https://copr.fedorainfracloud.org/coprs/managerforlustre/lustre-client/repo/epel-7/managerforlustre-lustre-client-epel-7.repo && \
+sudo yum-config-manager --add-repo https://downloads.hpdd.intel.com/public/e2fsprogs/latest/el7/ && \
+sudo sed -i -e "1d" -e "2s/^.*$/[e2fsprogs]/" -e "/baseurl/s/,/%2C/g" -e "/enabled/a gpgcheck=0" /etc/yum.repos.d/downloads.hpdd.intel.com_public_e2fsprogs_latest_el7_.repo && \
+sudo yum clean metadata && \
+sudo yum -y install lustre-client && \
+sudo reboot' c1 c2
```

Comment:
Rebooting the clients should not be necessary.  Please open a ticket if is.

Reply:
Removed reboot line